types/args/index.d.ts

```diff
@@ -0,0 +1,72 @@
+// Type definitions for args 3.0
+// Project: https://github.com/leo/args#readme
+// Definitions by: Slessi <https://github.com/Slessi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const c: args;
+export = c;
+
+interface args {
```

Comment:
You could just directly export `const`s and `function`s instead of `export =`-ing an object.